docs/en/File-System-API.md

```diff
@@ -155,34 +145,21 @@ Tachyon supports custom policies, so you can also develop your own policy approp
 workload. Note that a default policy must have an empty constructor. And to use ASYNC_THROUGH write
 type, all the blocks of a file must be written to the same worker.
 
-### Opening a TachyonFile
-
-Operations on an already existing file require its `TachyonFile` handle. The general contract of
-the filesystem client is that operations creating new files expect `TachyonURI` while operations on
-existing files expect `TachyonFile`. The `TachyonFile` handle is not a lock; other clients may
-still delete or rename the file. However, for renames, the handle will still be valid and reference
-the same file. A lock is used internally to guarantee atomicity of concurrent operations and also to
-prevent deletion of files that are opened for reading. Below is an example of how to obtain a
-`TachyonFile` handle:
+### Accessing an existing file in Tachyon
 
-```java
-TachyonFileSystem tfs = TachyonFileSystemFactory.get();
-// Assuming "/myFile" already exists
-TachyonURI path = new TachyonURI("/myFile");
-TachyonFile file = tfs.open(path);
-```
+All operations on existing files or directories require the user to specify the `TachyonURI`.
+With the TachyonURI, the user may use any of the methods of `FileSystem` to access the resource.
 
 ### Reading Data
 
-After obtaining a `TachyonFile` handle, the user may modify the file metadata or get an input stream
-to read the file. For example:
+After creating a `TachyonURI`, the user may modify the file metadata (for example, ttl or pin state
```

Comment:
This implies that all the user can do is modify file metadata or get an input stream. We should either be more specific and cover everything they can do, or more general and say that the `TachyonURI` is used to perform Tachyon FileSystem operations.